src/main/java/uk/gov/cshr/report/service/IdentityService.java

```diff
@@ -46,4 +58,22 @@ public Map<String, Identity> getIdentitiesMapForLearners(String learnerIds) {
                 .build(new HashMap<>());
         return httpService.getMap(uri, Identity.class);
     }
+
+    @Cacheable("service-token")
+    public OAuthToken getCachedOAuthServiceToken() {
```

Comment:
Report service doesn't use client tokens, so this can be removed

Reply:
`getCachedOAuthServiceToken()` seems to be used in [Interceptor](https://github.com/Civil-Service-Human-Resources/lpg-report-service/blob/51e5b9030e580812779d06d65434845929032429/src/main/java/uk/gov/cshr/report/service/auth/RestTemplateOAuthInterceptor.java#L22)